### DISCUSSION TITLE
Can't access benefits page

### DISCUSSION THREAD

**Discussion by vipulbhj** [community/community#38134] [GitHub Education]
<img width="1482" alt="Screenshot 2022-11-03 at 12 58 15 PM" src="https://user-images.githubusercontent.com/33618798/199667315-880e6c79-efc5-4b45-8ba3-24e9822f9e22.png">

Possibly some bad configuration, worst cause, some wider issue possible 😄

**Comment by ossanna16** [comment]
Hi there @vipulbhj and welcome to our community! Thank you for reporting this 🙂
